.github/scripts/cmake/FindThreads.cmake

```diff
@@ -0,0 +1 @@
+set(Threads_FOUND TRUE)
```

Comment:
I think this is probably left over from an earlier abandoned approach to get the Qt version, since I remember something trying to look for Threads and failing, and adding this to get it to succeed. Nothing in the Qt version cmake file looks to try to find dependencies any more so it shouldn’t be needed.

Reply:
This has been removed.